optimade/adapters/structures/utils.py

```diff
@@ -56,7 +58,7 @@ def scaled_cell(
     for i in range(3):
         vector = np.cross(cell[(i + 1) % 3], cell[(i + 2) % 3]) / volume
         scale.append(tuple(vector))
-    return tuple(scale)
+    return tuple(scale)  # type: ignore[return-value]
```

Comment:
The real problem here is that you cast the vector into a tuple. 
If you change line 60 to: `scale.append(Vector3D(vector))` you should not need `# type: ignore[return-value`

Reply:
When I use pre-commit run mypy --all-files 
I still get an error here so the # type: ignore[return-value] should remain here.